src/new-client/src/app/site/[site]/users/manage/assign-roles-form.tsx

```diff
@@ -40,64 +46,38 @@ const AssignRolesForm = ({
-        className={`nhsuk-form-group ${errors.roles ? 'nhsuk-form-group--error' : ''}`}
+      <FormGroup
+        error={
+          errors.roles
+            ? 'You have not selected any roles for this user'
+            : undefined
+        }
+        legend="Roles"
       >
-        <fieldset className="nhsuk-fieldset">
-          <legend className="nhsuk-fieldset__legend nhsuk-fieldset__legend--m">
-            <h1 className="nhsuk-fieldset__heading">Roles</h1>
-          </legend>
-          {errors.roles && (
-            <span className="nhsuk-error-message">
-              <span className="nhsuk-u-visually-hidden">Error:</span> You have
-              not selected any roles for this user
-            </span>
-          )}
-          <div className="nhsuk-checkboxes">
-            {roles.map(r => (
-              <div key={r.id} className="nhsuk-checkboxes__item">
-                <input
-                  id={r.id}
-                  type="checkbox"
-                  className="nhsuk-checkboxes__input"
-                  value={r.id}
-                  {...register('roles', { required: true })}
-                />
-                <label
-                  htmlFor={r.id}
-                  className="nhsuk-label nhsuk-checkboxes__label"
-                >
-                  {r.displayName}
-                </label>
-                <div
-                  className="nhsuk-hint nhsuk-checkboxes__hint"
-                  id="nationality-1-item-hint"
-                >
-                  {r.description}
-                </div>
-              </div>
-            ))}
-          </div>
-        </fieldset>
-      </div>
+        <CheckBoxes>
+          {roles.map(r => (
+            <CheckBox
+              id={r.id}
+              label={r.displayName}
+              hint={r.description}
+              key={`checkbox-key-${r.id}`}
+              value={r.id}
+              {...register('roles', { required: true })}
+            />
+          ))}
+        </CheckBoxes>
+      </FormGroup>
 
-      <div style={{ marginTop: '20px' }}>
-        <div className="nhsuk-navigation">
-          <button
-            type="submit"
-            aria-label="save user"
-            className="nhsuk-button nhsuk-u-margin-bottom-0"
-          >
-            Confirm and save
-          </button>
-          <button
-            type="button"
-            aria-label="cancel"
-            className="nhsuk-button nhsuk-button--secondary nhsuk-u-margin-left-3 nhsuk-u-margin-bottom-0"
-            onClick={cancel}
-          >
-            Cancel
-          </button>
+      <div className="nhsuk-grid-row">
+        <div className="nhsuk-grid-row">
+          <div className="nhsuk-grid-column-one-half">
+            <Button type="submit">Confirm and save</Button>
+          </div>
+          <div className="nhsuk-grid-column-one-half">
+            <Button styleType="secondary" onClick={cancel}>
+              Cancel
+            </Button>
+          </div>
```

Comment:
This styling isn't perfect - the two `one-half` columns cause more spacing between the buttons than I'd like. It's okay for now though, and I'd like to try to avoid writing custom inline styles if we can get away with it. Is it okay to leave this as it is for now and make a styling ticket for later?